Makefile

```diff
@@ -56,8 +56,11 @@ mod-tidy:
 test: vendor
 	CGO_ENABLED=0 go test $$(go list ./... | grep -v generated)
 
-$(LINTERS): %: vendor/bin/gometalinter %-bin vendor
-	PATH=`pwd`/vendor/bin:$$PATH gometalinter --tests --disable-all --vendor \
+install-gometalinter:
```

Comment:
`make vendor/bin/gometalinter` should do this -- I don't think we need this target, and the deps on `$(LINTERS)` shouldn't change